assets/model-fitting-logs.txt

```diff
@@ -0,0 +1,166 @@
+Training started - epochs: 5 - learning_rate: 0.001 - batch_size: 128 - training_examples: 508437 - classes: 10

```

Comment:
just curious what's this file for?

Reply:
It's just a record of how the model performed with metrics like precision, recall and f1. I thought it would be good to review how good the model actually is.